src/core/commands.mjs

```diff
@@ -732,6 +732,17 @@ export async function ToggleFullscreen (sender, data) {
 }
 
 
+export async function Fullscreen (sender, data) {
+  const window = await browser.windows.get(sender.tab.windowId);
+  
+  await browser.windows.update(sender.tab.windowId, {
+    state: 'fullscreen'
+  });
+  // confirm success
+  return true;
+}
```

Comment:
Please have a look at the `MaximizeWindow` command implementation.
If the window is already in fullscreen mode the function should not return true.